lib/datadog/appsec/configuration.rb

```diff
@@ -66,9 +66,7 @@ def obfuscator_value_regex=(value)
         end
 
         def [](key)
-          found = @instruments.find { |e| e.name == key }
-
-          found.options if found
+          @instruments.find { |e| e.name == key }
         end
```

Comment:
Would it make sense to remove this method? It's not like anything was using it, as otherwise, it would've broken, and if we're changing its semantics, we might as well.